engine/include/cubos/engine/renderer/viewport.hpp

```diff
@@ -0,0 +1,26 @@
+/// @file
+/// @brief Component @ref cubos::engine::Viewport.
+/// @ingroup renderer-plugin
+
+#pragma once
+#include <glm/glm.hpp>
+
+#include <cubos/core/reflection/reflect.hpp>
+
+namespace cubos::engine
+{
+
+    /// @brief Component which defines parameters of a viewport.
```

Comment:
Would be nice to explain here what the component is used for and what it changes in an entity with a camera.